src/assets/icons/confused-face.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
The icon is the correct canvas size, but visually the face looks a lot bigger than our other icons. It should be sized within its 24 x 24 canvas so that it appears in proportion to adjacent icons. It should not feel taller than the heart, pencil or envelope for example. 

<img width="986" alt="Screen Shot 2022-07-18 at 1 19 02 PM" src="https://user-images.githubusercontent.com/69633/179610137-e940edd5-df53-4ff4-905e-550efedc81c2.png">